ds3-cli-certification/src/test/java/com/spectralogic/ds3cli/certification/Certification_Test_8.java

```diff
@@ -0,0 +1,403 @@
+        }
+    }
+
+    /**
+     * 8.4: Manually change job priorities
+     */
+    @Test
+    public void test_8_4_change_priorities() throws Exception {
+        final String testDescription = "8.4: Manually change job priorities";
+        final String bucketName = CertificationUtil.getBucketName(testDescription);
+        boolean success = false;
+        String jobId = "";
+
+        OUT.startNewTest(testDescription);
+        OUT.insertLog("Start Bulk Job with single object, priority LOW");
+        try {
+            //create the Bucket
+            Util.createBucket(client, bucketName);
+
+            //create PUT job but don't start transfer
+            jobId = CertificationUtil.createPutJob(client, bucketName, Priority.LOW);
+
+            // show starting ("LOW");
+            final String getJobCmd = "--http -c get_job -i " + jobId;
+            final CommandResponse getInitialJobResponse = Util.command(client, getJobCmd);
+            OUT.insertCommand(getJobCmd, getInitialJobResponse.getMessage());
+            assertTrue(getInitialJobResponse.getMessage().contains("| Priority: LOW |"));
+
+            // set priority to URGENT
+            OUT.insertLog("Set priority to URGENT");
+            final String modifyJobPriorityUrgentCmd = "--http -c modify_job --priority URGENT -i " + jobId;
+            final CommandResponse modifyJobResponse = Util.command(client, modifyJobPriorityUrgentCmd);
+            OUT.insertCommand(modifyJobPriorityUrgentCmd, modifyJobResponse.getMessage());
+            assertTrue(modifyJobResponse.getMessage().contains("| Priority: URGENT |"));
+            success = true;
+
+        } catch (final Exception e) {
+            LOG.info("Exception in {}", testDescription, e );
+        } finally {
+            OUT.finishTest(testDescription, success);
+            CertificationUtil.deleteJob(client, jobId);
+            Util.deleteBucket(client, bucketName);
+            assertTrue(success);
+        }
+    }
+
+
+    /**
+     * 8.5: Eject a bucket or tape
+     */
+    @Test
+    public void test_8_5_eject_bucket() throws Exception {
+        final String testDescription = "8.5: Eject Bucket";
+        boolean success = false;
+        String barcode = "";
+        OUT.startNewTest(testDescription);
+        try {
+            // Get All Tapes -- once for display
+            final String getTapesArgs = "--http -c get_tapes";
+            Util.command(client, getTapesArgs);
+
+            // Get a normal tape
+            final GetTapesSpectraS3Response response = client.getTapesSpectraS3(new GetTapesSpectraS3Request());
+
+            final Collection normalTapes = CollectionUtils.filter(response.getTapeListResult().getTapes(),
+                    tape -> ((Tape) tape).getState() == TapeState.NORMAL
+                         || ((Tape) tape).getState() == TapeState.FOREIGN);
+            assertFalse(normalTapes.isEmpty());
+            barcode = ((Tape) normalTapes.iterator().next()).getBarCode();
```

Comment:
This cast shouldn't be needed if the collection is updated to specify its type using a generic.